### DISCUSSION TITLE
toggleVisible是否可以支持批量的显示和隐藏节点

### DISCUSSION THREAD

**Discussion by hysma** [antvis/X6#1846] [Q&A] [answered]
目前在大量使用react类型的节点时，当对一个大的分支进行折叠展开，渲染卡顿，我看了下代码，toggleVisible是对单个节点进行显隐处理，所以想问一下是否可以进行批量显示和隐藏操作

**Comment by NewByVector** [comment, marked as answer]
如果有性能问题，可以遍历节点使用样式 `display:none` 来隐藏，后面我们会优化这块逻辑。
